### DISCUSSION TITLE
メンバーについて

### DISCUSSION THREAD

**Discussion by kojira** [nostr-jp/nips-ja#126] [General]
アクティビティのないメンバーが残っていますが、プライベートリポジトリがあるため、プライベートリポジトリの内容が漏洩した時に疑われたくないという意見がありました。

対処案として、一年以上なんのアクティビティもないメンバーはメンバーから削除するなどが考えられますがいかがでしょうか？

現状プライベートリポジトリは技術書典関連だと思うので分離するという手もあると思います。
技術書典は四谷ラボ名義でやっていて内容もNostrだけではなくなっているので、四谷ラボリポジトリに移管する方が自然な気もします。

**Comment by SnowCait** [comment]
現状全メンバーがプライベートリポジトリにアクセスできる設定になっています。
![image](https://github.com/user-attachments/assets/93566c6d-17bb-4a38-bb0d-c8080f046b57)

ここの権限を None にしておけばプライベートリポジトリにはアクセスできなくなると思われます。
![image](https://github.com/user-attachments/assets/1411824e-1123-4f40-b781-e3e3a77b86f5)

**Comment by SnowCait** [comment]
セキュリティという意味では 2FA も必須にしておいた方が良さそうです。

**Comment by akiomik** [comment]
> 現状プライベートリポジトリは技術書典関連だと思うので分離するという手もあると思います。
技術書典は四谷ラボ名義でやっていて内容もNostrだけではなくなっているので、四谷ラボリポジトリに移管する方が自然な気もします。

+1

**Comment by ROBO358** [comment]
技術書典のリポジトリが移管されたため @kojira -sanの当初の懸念は解消されている状態かと思います。そのため、このままクローズで問題ないと思います。

![image](https://github.com/user-attachments/assets/ae39e1ae-da57-4343-9579-68582865cee8)

**Comment by kojira** [comment]
くろーずします
